tutorials/machine_learning/index.md

```diff
@@ -0,0 +1,140 @@
+\addtogroup tutorial_ml
+
+
+## Table of contents
+- [Basic TMVA tutorials](\ref basic)
+   - [Training](\ref training)
+   - [Applications](\ref application)
+   - [Others](\ref other)
+- [Cross validation](\ref cross_val)
+- [New TMVA interfaces](\ref new_interface)
+   - [RBDT](\ref rbdt)
+- [Deep learning in TMVA](\ref deep_learing)
+- [TMVA Keras tutorials](\ref keras)
+- [TMVA PyTorch tutorials](\ref pytorch)
+- [Interference with SOFIE](\ref interference)         
+- [RBatchGenerator](\ref rbatchgen)
+
+
+\anchor basic
+## Basic TMVA tutorials
+
+
+\anchor training
+### Training
+
+| **Tutorial** | **Description** |
+|--------------|-----------------|
+| TMVAMinimalClassification.C |  Minimal self-contained example for setting up TMVA with binary classification. |
+| TMVAMulticlass.C | This macro provides a simple example for the training and testing of the TMVA multiclass classification. |
```

Comment:
Every line in the table starts with "This macro": Is it really necessary? For example "This macro provides a simple example..." --> "A simple example..."